src/main.ts

```diff
@@ -8,15 +8,14 @@ import { AppModule } from './app.module'
 const open = require('open')
 
 awsConfig.update({
-  accessKeyId: 'foobar',
-  secretAccessKey: 'foobar',
-  region: 'localhost',
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: process.env.LOCALSTACK_REGION,
   logger: process.stdout,
 })
 
-const DEV_PORT = 10_007
-const isProd = process.env.NODE_ENV === 'prod'
-const serverPort = isProd ? 0 : DEV_PORT
+const appPort = parseInt(process.env.APP_PORT ?? '3000', 10);
```

Comment:
I would do it this way
```suggestion
const appPort = Number(process.env.APP_PORT) ?? 3000;
```